### PR TITLE
Signup: Try using a Gridicon in place of the "Continue" label

### PR DESCRIPTION
**Pull request by shaunandrews**
Before:
image

After:
image

### CODE REVIEW

File: client/signup/steps/site-information/index.jsx [hunk 1]
```diff
@@ -119,7 +120,7 @@ export class SiteInformation extends Component {
 
 	renderSubmitButton = () => (
 		<Button primary type="submit" onClick={ this.handleSubmit }>
```
Comment (ramonjd) [inline client/signup/steps/site-information/index.jsx]: Seeing as we're ridding ourselves of the text, maybe we could add some text helper attributes on the button?

`title` for the tooltip and `aria-label` for [accessibility](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute).

```
<Button title={ this.props.translate( 'Continue' ) } aria-label={ this.props.translate( 'Continue' ) } primary type="submit" onClick={ this.handleSubmit }>
```

File: client/signup/steps/site-topic/form.jsx [hunk 2]
```diff
@@ -82,7 +83,7 @@ class SiteTopicForm extends Component {
 							autoFocus={ true } // eslint-disable-line jsx-a11y/no-autofocus
 						/>
 						<Button type="submit" disabled={ isButtonDisabled } primary>
-							{ translate( 'Continue' ) }
+							<Gridicon icon="arrow-right" />
```
Comment (ramonjd) [inline client/signup/steps/site-topic/form.jsx]: Same as https://github.com/Automattic/wp-calypso/pull/31813/files#r269831352

File: client/signup/steps/site-information/index.jsx (hunk 1 above)
Comment (shaunandrews) [reply client/signup/steps/site-information/index.jsx]: NIce. Added.

**Comment by ramonjd** [comment]
I just updated the snapshot and rebased 👍 

btw, on Calypso Slack:

> desktop tests are failing for reasons we can’t control. feel free to ignore those failures for the time being.

**Pull request merged**